package.json

```diff
@@ -4,11 +4,11 @@
   "scripts": {
     "codegen": "graph codegen",
     "build": "graph build",
-    "test": "graph test",
+    "test": "mustache config/mainnet.json subgraph.yaml.mustache > subgraph.yaml && graph test",
```

Comment:
Can you create a sub command for this?
Like: `generateConfigs` or something like that